bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -428,6 +437,11 @@ pub struct CopyEvent {
     pub rw_counter_start: RWCounter,
     /// Represents the list of bytes related during this copy event
     pub copy_bytes: CopyBytes,
+    /// Represents transaction access list (EIP-2930), if copy data type is
+    /// address, the first item is access list address and second is zero, if
+    /// copy data type is storage key, the first item is access list address and
+    /// second is access list storage key.
+    pub access_list: Vec<(Address, Word)>,
```

Comment:
Since `AccessListAddress` and `AccessListStorageKey` are different copy data types and are also handled differently in the copy circuit, I would prefer something like this:
```suggestion
    pub access_list_address: Option<Address>,
    pub access_list_storage_key: Option<Word>,
```

I think we can also get rid of the `Default` impl (which leads to the `..Default::default()` usage).

Reply:
Do you mean I should fix to `acess_list_addresses: Vec<Address>` and `access_list_storage_keys: Vec<Word>`?
But the addresses and storage keys are related as [this test case](https://github.com/scroll-tech/zkevm-circuits/blob/56c432a4d49dd5ebfbcb71f115e99923fe4cfe9b/zkevm-circuits/src/copy_circuit/test.rs#L268) I added, there is 1-address corresponding to multiple storage keys. And when assigning copy table, an row is assigned as address to `value` and storage key to `value_prev`.
If we fix to two Vec for addresses and storage keys, they must have same length (and seems not related).
What do you think? Thanks.

Reply:
ah okay, this seems fair. We don't need to change this then. The current approach is good!